Subject: [PATCH] bootanimation: Call eglReleaseThread in threadLoop

Add eglReleaseThread call in the threadLoop function to
ensure that the thread properly cleans up its resources
before exiting.

diff --git a/cmds/bootanimation/BootAnimation.cpp b/cmds/bootanimation/BootAnimation.cpp
index 8f361ce..44b432e 100644
--- a/cmds/bootanimation/BootAnimation.cpp
+++ b/cmds/bootanimation/BootAnimation.cpp
@@ -348,6 +348,7 @@
     mFlingerSurface.clear();
     mFlingerSurfaceControl.clear();
     eglTerminate(mDisplay);
+    eglReleaseThread();
     IPCThreadState::self()->stopProcess();
     return r;
 }
